components/esp_websocket_client/examples/target/pytest_websocket.py

```diff
@@ -26,11 +26,18 @@ def get_my_ip():
     return IP
 
 
-class WebsocketTestEcho(WebSocket):
+def is_binary_strict(data):
+    return all(x in (0, 1) for x in data)
```

Comment:
Is this check supposed to test if  the received data is binary? Not sure if it's intended but it seems that it checks if the data contained only `'1'` and `'0'` as characters (so the ascii values of 30 and 31 dec)